lib/matplotlib/widgets.py

```diff
@@ -2013,8 +2017,8 @@ def on_select(min: float, max: float) -> Any
         The direction along which to draw the span selector.
 
     minspan : float, default: 0
-        If selection is less than or equal to *minspan*, do not call
-        *onselect*.
+        If selection is less than or equal to *minspan* and larger than 0,
+        the selection is removed (when already existing) or not completed.
```

Comment:
Isn't the selection also removed if the selection is zero?

Reply:
Done in e161b29.

Reply:
I meant that this sentence of the docs needed to be updated, no?  (at least removing the "larger than 0"?)

Reply:
Yes, that's correct - done in c7fb140.